### ISSUE TITLE
vxlan: vtep interface MTU needs to be configurable

### ISSUE THREAD

**Issue by leodotcloud** [rancher/rancher#8411]
Even though the MTU is configurable from the catalog and we use it to set the MTU of the `eth0` interface, it also needs to be set on the `vtep` interface.

Reference: https://github.com/rancher/rancher-net/blob/v0.10.0/backend/vxlan/vxlan.go#L21

**Comment by niusmallnan** [comment]
@leodotcloud 
I wonder why the vtep mtu was 1400. What was the history behind this?

**Comment by leodotcloud** [comment]
@niusmallnan when I was initially working on this stuff, there was a confusion about overhead being 50/98, so to be on the safe side I started with 1400. But now we know 50 is the correct overhead for VXLAN. All this is assuming an MTU of 1500 on the primary network interface.

**Comment by leodotcloud** [comment]
Fixed in `vxlan: v0.2.1` with image: `rancher/net:v0.11.9`

**Comment by sangeethah** [comment]
Tested with vxlan: v0.2.1 with image: rancher/net:v0.11.9. 

Created a template from "cattle" with ipsec disabled and vxlan enabled with MTU set to 9000.

mtu on the vtep interface gets set to configured value. 

```
root@15aa8c5bd8bd:/# ip addr
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group defaul
t qlen 1
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host
       valid_lft forever preferred_lft forever
3: eth0@if13: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc noqueue state UP
group default
    link/ether 02:05:d2:12:36:0a brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 10.42.127.103/16 scope global eth0
       valid_lft forever preferred_lft forever
    inet6 fe80::38c2:5cff:fefd:dc80/64 scope link
       valid_lft forever preferred_lft forever
4: vtep1042: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc noqueue state UNKN
OWN group default
    link/ether 0e:00:0a:2a:7f:67 brd ff:ff:ff:ff:ff:ff
    inet6 fe80::c00:aff:fe2a:7f67/64 scope link
       valid_lft forever preferred_lft forever
root@15aa8c5bd8bd:/#
```
